compiler/syscall/jasmin_syscall.h

```diff
@@ -2,7 +2,7 @@
 #ifndef JASMIN_SYSCALL
 #define JASMIN_SYSCALL
 /* FIXME this need xlen to be Uptr */
-unsigned char * __jasmin_syscall_randombytes__(unsigned char *x, uint64_t xlen)
+uint8_t* __jasmin_syscall_randombytes__(uint8_t* x, uint64_t xlen)
 asm("__jasmin_syscall_randombytes__");
```

Comment:
What’s the meaning of this `asm` stuff?

Reply:
I think the goal is to ensure that the function name is preserved. Maybe this was an issue in some testing environment (but I don't precisely recall the exact reason nor wrote any notes about it).

https://gcc.gnu.org/onlinedocs/gcc/Asm-Labels.html#Asm-Labels

https://clang.llvm.org/docs/AttributeReference.html#asm

Reply:
Thanks for the explanations & links.